CMakeLists.txt

```diff
@@ -115,7 +116,11 @@ endif()
 # Compile flags
 #
 
-set(CMAKE_CXX_STANDARD 11)
+if(LLAMA_BUILD_GRPC_SERVER)
+    set(CMAKE_CXX_STANDARD 11)
+else()
+    set(CMAKE_CXX_STANDARD 14)
```

Comment:
Move this inside the `grpc-server/CMakeLists.txt`

Reply:
yea, thats what i ment.